src/tools/rbd/action/Kernel.cc

```diff
@@ -153,6 +153,39 @@ static int do_kernel_showmapped(Formatter *f)
   return r;
 }
 
+static void print_error_description(const char *poolname, const char *imgname,
+				    const char *snapname, int maperrno)
+{
+  int r;
+  uint8_t oldformat;
+  librados::Rados rados;
+  librados::IoCtx ioctx;
+  librbd::Image image;
+
+  if (maperrno == ENOENT)
+    return;
```

Comment:
I excluded -ENOENT as its pretty self explanatory. No worries, I'll have that too.